src/Tracking/Tag.php

```diff
@@ -254,9 +254,15 @@ public function track_event( string $event_name, Data $data ) {
 		 * If the cart redirect is enabled, force add the event to the deferred events list
 		 * because if redirect is enabled, the cart page will be reloaded and the event will get lost.
 		 */
-		$is_redirect    = 'yes' === get_option( 'woocommerce_cart_redirect_after_add' );
+		$is_redirect = 'yes' === get_option( 'woocommerce_cart_redirect_after_add' );
+		/**
```

Comment:
This is not a doc block we should be using regular comment here. See https://arc.net/l/quote/ucuaptwj

```suggestion
		/*
```